Sources/Tabs/ProfileCoordinator.swift

```diff
@@ -36,4 +40,26 @@ final class ProfileCoordinator: NSObject,
     func updateToken(_ tokenHolder: TokenHolder) {
         baseVc?.updateToken(tokenHolder)
     }
+
+    func openSignOutPage() {
+        let navController = UINavigationController()
+        let vm = SignOutPageViewModel(analyticsService: analyticsCenter.analyticsService) { [unowned self] in
+          do {
+            analyticsCenter.analyticsPreferenceStore.hasAcceptedAnalytics = false
+            analyticsCenter.analyticsService.denyAnalyticsPermission()
+            // TODO: DCMAW-8933 will handle sign out error scenarios
+            try? userStore.clearTokenInfo()
+            try? userStore.secureStoreService.delete()
+            userStore.shouldPromptForAnalytics = true
+              root.dismiss(animated: false) {
+                  self.finish()
+              }
+          } catch {
+            print(error.localizedDescription)
+          }
```

Comment:
Getting a warning about no errors being caught here, we're using `try?` above which is why, are we sure we want to use that rather than `try`?

Reply:
Using `try? `right now ignores the secure store errors.  This is useful as running on simulator will throw those errors.  There is another ticket where we will be handling those errors, so I think it is ok to leave as `try?` for now.  I asked Joe to add a TODO referencing the ticket for handling the errors.  When we implement that ticket, we'll switch to `try`

Reply:
Got ya, makes sense 👍